lib/test.js

```diff
@@ -370,7 +370,7 @@ function error(err, msg, extra) {
     this._assert(!err, {
         message : defined(msg, String(err)),
         operator : 'error',
-        actual : err,
+        error : err,
```

Comment:
where is this field being read?

Reply:
Here, I believe:

https://github.com/substack/tape/blob/a1e8f7e21cc7831cf5a857b00627b5dbcdd68aea/lib/results.js#L164

Reply:
Forgive me, that was imprecise. What I should have said was that field is read off of the `opts` object here:
https://github.com/substack/tape/blob/a1e8f7e21cc7831cf5a857b00627b5dbcdd68aea/lib/test.js#L224
and that line populates the `error` field on the `'result'` event, which itself is read in the location I linked above.